### PR TITLE
Bugs/ap 4 bugfixes

### PR DESCRIPTION
**Pull request by willmcb**
# What: AP-4 bug fixes + minor changes

There were some minor changes that needed to be made to AP-4 before it could be signed off which Simon Meredith identified: 

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4) see comment at bottom.

### In summary:

**Text updates** 
Heading text - What does your client want legal aid for?
Input heading text - Search for legal proceedings
Hint text - For example by category of law, matter type or legal proceeding.

**Home button position and link**
Home button needed to be realigned so that it was closer to top border.

**Clear search**
Had to be moved slightly to align better with search-box, font-size also needed changing.

**No results**
Added 'No results found.' action if the user entered proceeding text in the input that's not available.

### CODE REVIEW

**Comment by willc-work** [comment]
You need to rebase with master

**Comment by willmcb** [comment]
@willc-work had not updated local master before rebase - have now done so and rebased again.

File: app/assets/javascripts/civil_applications/civil_applications.js [hunk 1]
```diff
@@ -36,6 +40,9 @@ var civil_applications = {
             $('#proceeding-search').val("");
             $("#proceeding-list").empty();
         });
+    },
+    add_result_rows: function(rows){
```
Comment (lostie) [inline app/assets/javascripts/civil_applications/civil_applications.js]: **[not a blocker]** 💄 This function doesn't seem to be used.

Comment (willmcb) [reply app/assets/javascripts/civil_applications/civil_applications.js]: I have removed in https://github.com/ministryofjustice/laa-apply-for-legal-aid-provider-frontend/pull/44/commits/71e8b5f060272d0a364bc8fd95615b6db7bad595

**Pull request merged**